examples/cifar10/readme.md

```diff
@@ -52,7 +53,8 @@ parse_cifar10(data_dir_path + "/test_batch.bin",
 
 # Grid Search
 One of the most important hyperparameter in deep learning is learning rate. 
-To get stable and better result, let's try [grid search](https://en.wikipedia.org/wiki/Hyperparameter_optimization#Grid_search)
+To get stable and better result, let's try
+ [grid search](https://en.wikipedia.org/wiki/Hyperparameter_optimization#Grid_search)
```

Comment:
@Randl grid search is not a good idea for DL. Use random search instead. We should mention this paper here
http://www.jmlr.org/papers/volume13/bergstra12a/bergstra12a.pdf